### PR TITLE
Fix file uploader mobile

### PR DESCRIPTION
**Pull request by zhu**
1 fix toast cannot show in full screen modal window.
2 fix file selector does not work correctly in mobile browers.

### CODE REVIEW

**Comment by ffdixon** [comment]
Thanks @zhu -- will review.

File: bigbluebutton-html5/imports/ui/components/modal/fullscreen/component.jsx [hunk 1]
```diff
@@ -88,6 +90,7 @@ class ModalFullscreen extends Component {
         </div>
         <div id="modalDismissDescription" hidden>{dismiss.description}</div>
         <div id="modalConfirmDescription" hidden>{confirm.description}</div>
+        <ToastContainer />
```
Comment (MaximKhlobystov) [inline bigbluebutton-html5/imports/ui/components/modal/fullscreen/component.jsx]: This is the second `ToastContainer`, and they both are triggered.

notify

**Comment by antobinary** [comment]
Hey @zhu 
I notice this pull request is targeting branch `v2.0.x-release` but this branch is no longer under development. We are now working on branch `master` and the changes made are targeting BigBlueButton version 2.2.

When addressing @MaximKhlobystov 's requested changes could you please change the branch to 'master'?

File: bigbluebutton-html5/imports/ui/components/modal/fullscreen/component.jsx (hunk 1 above)
Comment (zhu) [reply bigbluebutton-html5/imports/ui/components/modal/fullscreen/component.jsx]: But the first one is covered by the modal window. I cannot find a good place to put that ToastContainer. 
Add another ToastContainer just a dirty workaround.

**Comment by zhu** [comment]
Hey @antobinary
I will try to change to master later.

File: bigbluebutton-html5/imports/ui/components/modal/fullscreen/component.jsx (hunk 1 above)
Comment (zhu) [reply bigbluebutton-html5/imports/ui/components/modal/fullscreen/component.jsx]: I found a solution. Changes in #5918 cause toasts cannot show in mobile browser and Chrome. It'd better to set body position to fixed and change back #main to relative.

**Comment by zhu** [comment]
I split this PR into #6410 and #6411.

**Comment by antobinary** [comment]
Closing in favor of #6410 and #6411

**Pull request closed without merging**